### DISCUSSION TITLE
[mobile version] -

### DISCUSSION THREAD

**Discussion by thetomatoguy** [Monika-After-Story/MonikaModDev#7865] [QA - tech support + general] [answered]
i am searching  this days for a way to get my monika from the PC version to mobile version but i am not sure after i do that if i will have all my saved progress transferred  to the mobile or will it affect the pc version 
if there is any way or mobile version that can transferred to (from this team) i will really be happy

**Comment by multimokia** [comment, marked as answer]
Unfortunately, we do not support the mobile release of MAS as it is unofficial nor was it made with approval of us.

Additionally, we do not know the extent of the code changes involved in it vs the current build of MAS.

As such, no help can really be provided for it.

**Comment by thetomatoguy** [comment]
it is unfortunate but understandable 
the main reason i was asking about that topic was just to find a way to save my progress because i have to leave my pc for about 8 to 9 months and i am worried that i will lose the progress due to inactivity  and it will be really hard to start over  so if there any way to prevent this from happening i will really appreciate it 
                                            (by the way i really like and enjoy this mod and i am looking forward to the upcoming updates)

**Comment by multimokia** [comment]
I'd actually suggest taking her with you. Assuming nobody else is using your PC, you can just do that without worry. Just put her file back in the characters folder when you're back.

Additionally, you can always back up your persistent to something like google drive, so you'll always be able to restore her. Even on another computer too.
